src/transformers/models/big_bird/modeling_flax_big_bird.py

```diff
@@ -1067,11 +1102,11 @@ def _bigbird_block_rand_mask_with_head(
                 # column indx start fromm plan_block_length[plan_idx-1] and ends at
                 # plan_block_length[plan_idx]
                 if plan_num_rand_blocks[plan_idx] > 0:
-                    rnd_r_cnt = int(np.sum(plan_num_rand_blocks[:plan_idx]))
-                    curr_r_cnt = int(np.sum(plan_num_rand_blocks[: plan_idx + 1]))
+                    rnd_r_cnt = int(sum(plan_num_rand_blocks[:plan_idx]))
```

Comment:
Think jnp.sum's here would be better too

Reply:
@sanchit-gandhi  I have used that in the first place, but while using it for example in line `1082` during `rand_attn` initialization, it requires casting `plan_num_rand_blocks` to `jnp.array` which results in the error that I wasn't able to fix:

```
jax._src.errors.TracerArrayConversionError: The numpy.ndarray conversion method __array__() was called on the JAX Tracer object Traced<ShapedArray(int32[])>with<DynamicJaxprTrace(level=1/0)>
           While tracing the function init_weights at /home/bearnard/Dev/transformers/src/transformers/models/big_bird/modeling_flax_big_bird.py:1640 for eval_shape, this value became a tracer due to JAX operations on these lines:

            operation a:i32[2] = convert_element_type[new_dtype=int32 weak_type=False] b
               from line /home/bearnard/Dev/transformers/src/transformers/models/big_bird/modeling_flax_big_bird.py:1083 (<listcomp>)
           See https://jax.readthedocs.io/en/latest/errors.html#jax.errors.TracerArrayConversionError
```

Reply:
That's weird! Went through quite a bit of the JAX code to see what they use, seems like `sum()` and `jnp.sum()` are both fine, so if `sum()` works lets stick with that.